package.json

```diff
@@ -1,11 +1,17 @@
 {
   "name": "fbp",
   "description": "Parser for the .fbp flow definition language",
-  "version": "1.4.0",
+  "version": "1.5.0",
```

Comment:
Please don't change version numbers in PRs, it very quickly creates merge conflicts. Better to add entry in CHANGES.md